plugins/view-mapped/src/view-mapped.ts

```diff
@@ -104,6 +120,23 @@ export function viewMappedPlugin() {
 
           await tick(80, 'plugins.viewMapped.almostDone')
 
+          // Check that all commits are completed before running this job
+          let hasUncompletedCommits = true
+          do {
+            const { data: commits } = await api.sheets.getSheetCommits(
```

Comment:
Is there any other way to trigger this other than polling? Can we listen for completed commits or do we need to make another mechanism to control the order of these event responses?